packages/amplify-cli/src/input-manager.ts

```diff
@@ -193,3 +193,9 @@ export function verifyInput(pluginPlatform: PluginPlatform, input: Input): Input
 
   return result;
 }
+
+function aliasArgs(argv: string[]) {
+  if (argv.length >= 4 && argv[2] === 'override' && argv[3] === 'project') {
+    argv[3] = 'rootStack';
```

Comment:
you can just use "root" - that should work as is since it's an alias already defined.

Reply:
I thought root is for something else if it's only for override then I can change it